### PR TITLE
[V3] convert nodes_compositing.py to V3 schema

### PR DESCRIPTION
**Pull request by bigcat88**
Nodes were tested after conversion:

Screenshot From 2025-10-03 11-04-48

These changes were required to make V3 nodes work, as `io.NodeOutput` expect `*args` and not tuple(was not possible to do `return io.NodeOutput(result)`

```python
result = (torch.stack(out_images), 1.0 - torch.stack(out_alphas))
return result
```
->
```python
return io.NodeOutput(torch.stack(out_images), torch.stack(out_alphas))
```

Git diff:

```
diff --git a/v3_object_info.json b/master_object_info.json
index 4b7579d..35fd591 100644
--- a/v3_object_info.json
+++ b/master_object_info.json
@@ -5271,46 +5271,40 @@
         "input": {
             "required": {
                 "source": [
-                    "IMAGE",
-                    {}
+                    "IMAGE"
                 ],
                 "source_alpha": [
-                    "MASK",
-                    {}
+                    "MASK"
                 ],
                 "destination": [
-                    "IMAGE",
-                    {}
+                    "IMAGE"
                 ],
                 "destination_alpha": [
-                    "MASK",
-                    {}
+                    "MASK"
                 ],
                 "mode": [
-                    "COMBO",
+                    [
+                        "ADD",
+                        "CLEAR",
+                        "DARKEN",
+                        "DST",
+                        "DST_ATOP",
+                        "DST_IN",
+                        "DST_OUT",
+                        "DST_OVER",
+                        "LIGHTEN",
+                        "MULTIPLY",
+                        "OVERLAY",
+                        "SCREEN",
+                        "SRC",
+                        "SRC_ATOP",
+                        "SRC_IN",
+                        "SRC_OUT",
+                        "SRC_OVER",
+                        "XOR"
+                    ],
                     {
-                        "default": "DST",
-                        "multiselect": false,
-                        "options": [
-                            "ADD",
-                            "CLEAR",
-                            "DARKEN",
-                            "DST",
-                            "DST_ATOP",
-                            "DST_IN",
-                            "DST_OUT",
-                            "DST_OVER",
-                            "LIGHTEN",
-                            "MULTIPLY",
-                            "OVERLAY",
-                            "SCREEN",
-                            "SRC",
-                            "SRC_ATOP",
-                            "SRC_IN",
-                            "SRC_OUT",
-                            "SRC_OVER",
-                            "XOR"
-                        ]
+                        "default": "DST"
                     }
                 ]
             }
@@ -5336,26 +5330,18 @@
             "IMAGE",
             "MASK"
         ],
-        "output_tooltips": [
-            null,
-            null
-        ],
         "name": "PorterDuffImageComposite",
         "display_name": "Porter-Duff Image Composite",
         "description": "",
         "python_module": "comfy_extras.nodes_compositing",
         "category": "mask/compositing",
-        "output_node": false,
-        "deprecated": false,
-        "experimental": false,
-        "api_node": false
+        "output_node": false
     },
     "SplitImageWithAlpha": {
         "input": {
             "required": {
                 "image": [
-                    "IMAGE",
-                    {}
+                    "IMAGE"
                 ]
             }
         },
@@ -5376,30 +5362,21 @@
             "IMAGE",
             "MASK"
         ],
-        "output_tooltips": [
-            null,
-            null
-        ],
         "name": "SplitImageWithAlpha",
         "display_name": "Split Image with Alpha",
         "description": "",
         "python_module": "comfy_extras.nodes_compositing",
         "category": "mask/compositing",
-        "output_node": false,
-        "deprecated": false,
-        "experimental": false,
-        "api_node": false
+        "output_node": false
     },
     "JoinImageWithAlpha": {
         "input": {
             "required": {
                 "image": [
-                    "IMAGE",
-                    {}
+                    "IMAGE"
                 ],
                 "alpha": [
-                    "MASK",
-                    {}
+                    "MASK"
                 ]
             }
         },
@@ -5418,18 +5395,12 @@
         "output_name": [
             "IMAGE"
         ],
-        "output_tooltips": [
-            null
-        ],
         "name": "JoinImageWithAlpha",
         "display_name": "Join Image with Alpha",
         "description": "",
         "python_module": "comfy_extras.nodes_compositing",
         "category": "mask/compositing",
-        "output_node": false,
-        "deprecated": false,
-        "experimental": false,
-        "api_node": false
+        "output_node": false
     },
     "RebatchLatents": {
         "input": {
```

### CODE REVIEW

**Comment by bigcat88** [comment]
+label: Core

**Comment by Kosinkadink** [comment]
Node authors notified, will merge on Wednesday Oct. 8th with the rest of the scheduled V3 PR batch.
